common/src/main/java/com/redhat/cloud/notifications/models/Event.java

```diff
@@ -8,24 +8,30 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.OneToMany;
+import javax.persistence.PrePersist;
 import javax.persistence.Table;
 import javax.persistence.Transient;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
 import java.util.Objects;
 import java.util.Set;
 import java.util.UUID;
 
+import static java.time.ZoneOffset.UTC;
 import static javax.persistence.CascadeType.REMOVE;
 
 @Entity
 @Table(name = "event")
-public class Event extends CreationTimestamped {
+public class Event {
 
     @Id
     @GeneratedValue
     private UUID id;
 
+    private Timestamp created;
```

Comment:
how this worked before?

Reply:
The `LocalDateTime created` field from `CreationTimestamped` was used but it required a date transformation with Postgres' `DATE()` function.